communities/UNAM-Engineering/community-impact-proposal.md

```diff
@@ -0,0 +1,109 @@
+
+
+### Community demographics
+  - **Size**: 42 Registered Members; 10 members sharing repos.
+  - **Age**: 18-30 years.
+  - **Main Backgrounds**: Computer Engineers, Computer Scientists.
+  - **Secondary Backgrounds**: Psicologists, Lawyers, etc.
+  - **Resources**:
+    - **People**: We have students we lots of expierience and freshmen as well.
+    Also we have some teachers who mainly work on open source.
+    - **Github Organization**: Our main platform it allows us to achieve our
+    goals. It allows discussiones, code shaing and information gathering.
+
+### Communication
+We usually communicate in private FB Messages or via Github Issues.
+
+### Culture
+We strongly believe that opening school projects gives others the posibility to
+learn from others. We're open to have a diverse community based on the will to learn
+and share. We know that students on the campus are developing projects and research 
+papers that should be proudly shown to others outside the university.
+
+
+## Proposal
+What we're looking for in the first month of the project is to have a well
+stablished structure on the Github Organization that would allow new commers
+to join the group/conversation easily and understand the collaboration method
+that we apply. Also would like to hold small meetups so the community starts
+to know each other and get to meet people in the industry.
+
+
+### Timing
+| Week |              Title              |                                                    Description                                                    |           Status            |
+|:----:|:-------------------------------:|:-----------------------------------------------------------------------------------------------------------------:|:---------------------------:|
+|   3  | Research Similar  Organizations | Look for similar organization on github that use organizations to  manage the group. Analyze what works for them. |    :white_square_button:    |
+|   4  | Implement Structure Changes     | After finishing the research on other organizations implement the structure they have in our GH organization.     |    :white_square_button:    |
+|   5  | Generate event calendar        | Publish an event calendar on the organization that includes on campus events and off campus events as well.        |    :white_square_button:    |
+|   6  | Push Changes & Feedback         | Push changes to GH Organization and receive feedback from  the members.                                           |    :white_square_button:    |
+
+### Community Demographics
+For these first steps will be focusing on Computer Engineers since they are
+the closest demographic with the availability that others enter as well.
+
+### Goals
+  - **Structure, structure, structure**.
+  - Event Planning.
+  - Improve accesibility for new commers.
+  - Recruit **Awesome** People.
+
+### Communication
+This shoud be discussed. Though Slack sounds as a tipical answer to communication
```

Comment:
Agreed. Slack is tricky, you need a very clear reason for people to come there.

For your group though, I think you've got that. People come to your group to get help and examples for their classwork. They come to YOU! That means you can set the terms of engagement, so if you say "this week, if you want help, it'll have to be in slack", you can make it work. Most groups don't have that dynamic, they want to make a community but the community doesnt' need them. Here they clearly need you